README.md

```diff
@@ -1,5 +1,5 @@
 
-
+https://math-magicians-gold.vercel.app/Calculator
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
```

Comment:
- [ ] As requested by the previous code reviewer, you should include all sections of the [readme template from microverse](https://github.com/microverseinc/readme-template) (only the FAQ is optional) but currently, you are missing the `Live Demo` section. :blush: Also your getting started section should be in detail showing visitors how to clone the project locally.
- [ ] As requested already, Why don't you remove all comments from your readme since these comments were just in the template you used to instruct you to do something, and after you are done you have to remove them from your readme to make it professional :smile:  for instance these comments :point_down: which should be removed 
    
    - _Write a message to encourage readers to support your project_
    - _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
    - _Mention all of the collaborators of this project._
    - _Give credit to everyone who inspired your codebase._
    - _Describe your project in 1 or 2 sentences._